features/step_definitions/your_results_steps.rb

```diff
@@ -0,0 +1,20 @@
+Given(/^that I have completed the first of the WPCC tool$/) do
+  # Step One - journey
+  step 'I am on step 1 of the WPCC homepage'
+  step 'I fill in the age, gender, salary and frequency fields'
+  step 'I click on "My employer makes contributions on part of my salary"'
```

Comment:
On my branch I will change this figures percentages to be different (one 1% and the other 2%). The reason: there was a bug that the test didn't catch. I just saying because there was a typo (a duplicated employee percent) that took me some time to figure it out that the problem were not in my branch.